clients/cli/Cargo.toml

```diff
@@ -18,3 +18,4 @@ chrono = "0.4"
 qr2term = "=0.2.1"
 hotwatch = "0.4.5"
 atty = "0.2"
+ptree = "0.4"
```

Comment:
what made you choose this dep? could be fine but when I look at the repo for signs of popularity i see that it goes to a gitlab with 1 star. widely used software is safer in a few regards than niche software.

Reply:
usually this will cause a change to the `.lock` file. it's a security mechanism, you want to check that bad boy in as well.